pkg/api/mlflow/api/request/artifact.go

```diff
@@ -7,10 +7,25 @@ type ListArtifactsRequest struct {
 	RunUUID string `query:"run_uuid"`
 }
 
+// GetArtifactRequest is for retrieving an individual artifact item.
+type GetArtifactRequest struct {
+	Path    string `query:"path"`
+	RunID   string `query:"run_id"`
+	RunUUID string `query:"run_uuid"`
+}
+
 // GetRunID returns Run ID.
 func (r ListArtifactsRequest) GetRunID() string {
 	if r.RunID != "" {
 		return r.RunID
 	}
 	return r.RunUUID
 }
+
+// GetRunID returns RunID if available, otherwise RunUUID.
+func (r GetArtifactRequest) GetRunID() string {
+	if r.RunID != "" {
+		return r.RunID
+	}
+	return r.RunUUID
+}
```

Comment:
just a small tiny change - can you please move it to be near original struct `GetArtifactRequest`?

Reply:
yep